scipy/stats/_multivariate.py

```diff
@@ -5066,3 +5067,161 @@ def rvs(self, size=1, random_state=None):
         method.__doc__, mhg_docdict_noparams)
     method.__doc__ = doccer.docformat(method.__doc__,
                                       mhg_docdict_params)
+
+
+class random_direction_gen(multi_rv_generic):
+    r"""A vector-valued random direction.
+
+    Return a random direction (unit vector).
+
+    The `dim` keyword specifies the dimension N.
+
+    Methods
+    -------
+    rvs(dim=None, size=1, random_state=None)
+        Draw random directions.
+
+    Parameters
+    ----------
+    dim : scalar
+        Dimension of directions.
+    seed : {None, int, np.random.RandomState, np.random.Generator}, optional
```

Comment:
If you want this to link to NumPy, it has to be `numpy` instead of `np`. If you write it all out on one line, there's a PEP8 issue. If you try to silence that, it shows up in the documentation (unless you add the comment after the whole docstring. So in some other stats functions we break it up like:
```suggestion
    seed : {None, int, `numpy.random.Generator`,
            `numpy.random.RandomState`}, optional

```